src/models/ITestType.ts

```diff
@@ -7,6 +7,10 @@ export interface ITestType {
     forVehicleSize: ForVehicleSize[] | null;
     forVehicleConfiguration: ForVehicleConfiguration[] | null;
     forVehicleAxles: number[] | null;
+    forVehicleCategory: ForVehicleCategory[] | null;
+    forVehicleClass: number[] | null;
```

Comment:
Are you sure this field is a number? In the API specs I see it defined as a string

Reply:
in the excel it is a number that's why i thought it is a number as in axles or wheels. will change, thanks

Reply:
done